app/controllers/results_controller.rb

```diff
@@ -10,6 +10,11 @@ def create
     respond_with(:profile, @result)
   end
 
+  def bulk_delete
+    Result.destroy(params[:result_ids]) if params[:result_ids]
```

Comment:
Wouldn't `current_user.results.destroy(params[:result_ids])` be sufficient to prevent hacking?